src/engine/Server.h

```diff
@@ -38,29 +38,27 @@ class Server {
  private:
   //! Initialize the server.
   void initialize(const string& indexBaseName, bool useText,
-                  bool usePatterns = true, bool usePatternTrick = true,
-                  bool loadAllPermutations = true);
+                  bool usePatterns = true, bool loadAllPermutations = true);
```

Comment:
The deeper question is:
Is there any benefit of the two functions (constructor + run), 
Or should the server just immediately start in the constructor.
Two-phase-initialization is always a bit fishy.